documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -127,3 +128,42 @@ If a placeholder `{}` is added to the message it will be replaced with streaming
 |`{"message": "Signal message at offset {}"}`
 
 |===
+
+
+=== Triggering an Incremental Snapshot
+*Available for MySQL, PostgresSQL, SQL Server and Db2 connectors*
+
+The type of the action is `execute-snapshot`.
+It will start an incremental snapshot operation up to the oldest available streamed event.
```

Comment:
Unfortunately I am really not sure what you want to express here. Could you please elaborte a bit more?

Reply:
I wanted to convey what is the end point of the incremental snapshot operation, this was my best guess lol and was hoping someone could help me on this

Reply:
I see. So no, this is not accurate. The incremental snapshot finds the largest PK of the table at start of the snapshot of that table and finishes when this point is reached.

Reply:
Will correct, thanks